wordpress-mobile/WordPress-iOS
# iOS 14 Today Widget: Small Size UI

Ref #15103
This adds the small size Today Widget for iOS 14

Notes: 
- this PR only adds the UI for the small size Widget.
- this PR also contain additional commits that fix provisioning profiles and allow to cut installable builds

Known issues:
- The content of the widget is just an example
- The medium size widget is not implemented yet
- The displayed number of views does not align with the designs yet, specifically the decimal separator does not currently encroach into the padding area
- The vertical spacing between title lines, title and "Today" and "Views" and views number is automatically set and not explicitly set to 8 pts for now.

To test:

- install the app on a device or simulator with iOS 14
- add a new widget, scroll down to find WordPress and then tap to add
- make sure the added widget is consistent with the designs in #15103. See also examples below
<p align=center>
<img width=250 height= 250 src=https://user-images.githubusercontent.com/34376330/96768084-063a1980-13a3-11eb-9a1f-d8c8e62ef546.png>  <img width= 250 height= 250 src=https://user-images.githubusercontent.com/34376330/96769491-d25ff380-13a4-11eb-9301-a2f8346c2edb.png>
</p>


PR submission checklist:
